eng/common/scripts/get-codeowners.ps1

```diff
@@ -80,16 +141,16 @@ function TestGetCodeOwner([string]$targetDirectory, [string]$codeOwnerFileLocati
   }
 }
 
-if($Test) {
+if ($Test) {
```

Comment:
Now that we are establishing a testing pattern we should probably follow that and move these tests outs into their own tests script. However we can do that as a follow-up if you like.

Reply:
I created an issue to track it:
- https://github.com/Azure/azure-sdk-tools/issues/5434